src/modules/universal-discovery/services/universal.discovery.service.js

```diff
@@ -195,8 +195,9 @@ export const findLocationsByParentLocationId = (
  * @param {Object} restInfo REST config hash containing: token and siteaccess properties
  * @param {String} query
  * @param {Function} callback
+ * @param {Integer} limit
  */
-export const findContentBySearchQuery = ({ token, siteaccess }, query, callback) => {
+export const findContentBySearchQuery = ({ token, siteaccess }, query, callback, limit) => {
```

Comment:
You should provide the default value for BC. `limit = QUERY_LIMIT`

Reply:
Now it's provided